src/components/Modal/Rating.ts

```diff
@@ -0,0 +1,99 @@
+import { Score } from "../../../types/starRating";
+import { SCORE_MESSAGES } from "../../constants/config";
+import Store from "../../store/store";
+
+const ALLOWED_RATINGS: Score[] = [2, 4, 6, 8, 10];
```

Comment:
상수가 타입을 대표하는 케이스라면, 이렇게 상수로부터 타입을 만들어서 관리포인트를 1개로 줄이는 것을 추천합니다~!

```suggestion
const ALLOWED_RATINGS = [2, 4, 6, 8, 10];
type Score = (typeof ALLOWED_RATINGS)[number];
```